exasol/slc_ci/lib/prepare_test_container.py

```diff
@@ -0,0 +1,34 @@
+from exasol.slc_ci.lib.ci_build_test_container import CIBuildTestContainer
+from exasol.slc_ci.lib.ci_prepare import CIPrepare
+from exasol.slc_ci.lib.ci_push_test_container import CIPushTestContainer
+from exasol.slc_ci.lib.get_build_config_model import get_build_config_model
+
+
+def prepare_test_container(
+    docker_user: str,
+    docker_password: str,
+    commit_sha: str,
+    ci_prepare: CIPrepare,
+    ci_build_test_container: CIBuildTestContainer,
+    ci_push_test_container: CIPushTestContainer,
+) -> None:
+    build_config = get_build_config_model()
+    ci_prepare.prepare()
+
+    ci_build_test_container.build_test_container(
```

Comment:
I think, we can skip the build and directly push, because the push should invoke the build

Reply:
Done